pkg/services/ngalert/api/api.go

```diff
@@ -152,6 +153,11 @@ func (api *API) RegisterAPIEndpoints(m *metrics.API) {
 		muteTimings:         api.MuteTimings,
 		alertRules:          api.AlertRules,
 	}), m)
+
+	api.RegisterHistoryApiEndpoints(NewStateHistoryApi(&HistorySrv{
```

Comment:
We aren't checking any flags, and instead just registering this endpoint straight up.

It doesn't need to wait on the Loki work to be completed. It works just fine for annotations too, you get a dataframe containing the annotation data. When using annotations, some query parameters are not supported, but you'll still get something back.